iphone/CoreApi/CoreApi/Bookmarks/MWMBookmarksManager.mm

```diff
@@ -640,6 +667,41 @@ - (void)updateBookmark:(MWMMarkID)bookmarkId
   }
 }
 
+- (void)moveBookmark:(MWMMarkID)bookmarkId
+           toGroupId:(MWMMarkGroupID)groupId {
+    auto const currentGroupId = self.bm.GetBookmark(bookmarkId)->GetGroupId();
+    auto editSession = self.bm.GetEditSession();
```

Comment:
Why is it needed as a separate variable here and below?

Reply:
Not needed, I thought it is your code style - I see such pattern everywhere in `MWMBookmarksManager.mm`.
Can be easily removed, no need for extra var here.